live-examples/html-examples/content-sectioning/css/header.css

```diff
@@ -0,0 +1,29 @@
+header.page-header {
+    background: no-repeat left/cover url(/media/examples/puppy-header-logo.jpg);
+    display: flex;
+    height: 120px;
+    min-width: 120px;
```

Comment:
I don't think we need this line at all.

Reply:
What very narrow width? At about 180px, it looks the same with or without this line, and I don't think it is expected to work at narrower widths than this (at this point you are getting 1-2 words per line, so it isn't very usable anyway).